src/Hooks.php

```diff
@@ -106,14 +124,29 @@ public static function onAPIFlowAfterExecute( ApiFlowBase $module ): void {
 			return;
 		}
 
+		self::rebuildForPage( $page );
+	}
+
+	/**
+	 * This method is used to (forcefully) update the semantic index after a write has been performed by Flow/SD.
+	 *
+	 * @param string $pageName The page to rebuild
+	 * @return void
+	 */
+	private static function rebuildForPage( string $pageName ): void {
 		$store = StoreFactory::getStore();
 		$store->setOption( Store::OPT_CREATE_UPDATE_JOB, false );
 
 		$dataRebuilder = new DataRebuilder( $store, ServicesFactory::getInstance()->newTitleFactory() );
-		$dataRebuilder->setOptions( new Options( [ 'page' => $page ] ) );
+		$dataRebuilder->setOptions( new Options( [ 'page' => $pageName ] ) );
 		$dataRebuilder->rebuild();
 	}
 
+	/**
+	 * This method is used to reserve the usernames of the system user(s) used by this extension
+	 *
+	 * @link onUserGetReservedNames
```

Comment:
This link links to the method itself.

```suggestion
	 * @link https://www.mediawiki.org/wiki/Manual:Hooks/UserGetReservedNames
```